app/Models/Contact/Contact.php

```diff
@@ -153,7 +153,7 @@ public function gender()
     /**
      * Get the activity records associated with the contact.
      *
-     * @return HasMany
+     * @return BelongsToMany
```

Comment:
Why? It really is a HasMany relationship.

Reply:
hmm perhaps I have the wrong version of master? for me at app/Models/Contact/Contact.php:160 I see 
`return $this->belongsToMany(Activity::class)->orderBy('date_it_happened', 'desc');`

`belongsToMany`

Reply:
perhaps the relationship is wrong not the docbock?

Reply:
must I fix these relationships?